### DISCUSSION TITLE
How can i identify the main components?

### DISCUSSION THREAD

**Discussion by feliperad** [MaxHalford/prince#156] [Q&A] [answered]
I'musing the famd with the load beers dataset, but I have one doubt: how can I identify the main components?

**Comment by MaxHalford** [comment]
You can access the `eigenvalues_summary` property of your `FAMD` instance.

**Comment by feliperad** [comment]
dear Max,

i'm familiar with the eigenvalues_summary, but it only shows the
eigenvalues, percent of variance (non and cumulative) of the desired number
of components; but what i'd like to know is: based on the data given, who's
the component 0? who's the component 1? And so on.

For instance, for the beers dataset, I'd like to see something like this:

component 1 - "alcohol by volume"
component 2 -  "final gravity"

--
Felipe Nunes Radtke
***@***.***


Em qui., 22 de jun. de 2023 às 04:12, Max Halford ***@***.***>
escreveu:

**Reply by MaxHalford** [reply, marked as answer]
> component 1 - "alcohol by volume"
> component 2 -  "final gravity"

These aren't components, they're the columns in your original dataset. I think what you want is the contribution of each column to every component, i.e. the `column_contributions_` property.
